### DISCUSSION TITLE
OnCreateCoreWebView2ControllerCompleted HRESULT 0x80080005

### DISCUSSION THREAD

**Discussion by JasonHeGit** [MicrosoftEdge/WebView2Feedback#3977] [Q&A]
**Description**
For a few customers, the HRESULT in OnCreateCoreWebView2ControllerCompleted is 0x80080005 which seems to be CO_E_SERVER_EXEC_FAILURE . Any idea what would cause this error? Thank you for any help.

**Version**
SDK: 1.0.864.35
Runtime:  Fixed distribution 91.0.864.64
Framework: Win32
OS: Windows 10

**Repro Steps**
Unable to duplicate internally

**Comment by sln162** [comment]
We have also collected this error, and the reason is unknown. It is the latest SDK and multiple initialization failure reports sent by the latest runtime 116.0.1938.62.

We checked and found that there have been multiple occurrences in the past few months. We tried to switch the UDF position when encountering 0x80080005, but it did not work.
